dev/com.ibm.ws.ejbcontainer.v32_fat/fat/src/com/ibm/ws/ejbcontainer/v32/fat/tests/PassivationTest.java

```diff
@@ -39,6 +42,13 @@ public class PassivationTest extends FATServletClient {
 
     private static boolean checkLogsAfterServerStop;
 
+    @ClassRule
+    public static RepeatTests r = RepeatTests.with(FeatureReplacementAction.EE7_FEATURES().forServers("com.ibm.ws.ejbcontainer.v32.fat.passivation")) //
```

Comment:
```suggestion
    public static RepeatTests r = RepeatTests.with(FeatureReplacementAction.NO_REPLACEMENT().forServers("com.ibm.ws.ejbcontainer.v32.fat.passivation")) //
```
Question: is it necessary to to add the `EE7_FEATURES()` replacement action when the server configuration has EE7 features already?